oak-run-commons/src/main/java/org/apache/jackrabbit/oak/fixture/SegmentTarFixture.java

```diff
@@ -346,6 +354,7 @@ public Oak[] setUpCluster(int n, StatisticsProvider statsProvider) throws Except
                     .withSegmentCacheSize(segmentCacheSize)
                     .withMemoryMapping(memoryMapping)
                     .withStrictVersionCheck(true)
+                    .withBinariesInlineThreshold(binariesInlineThreshold)
```

Comment:
Does it make sense to use this option in other methods where ```FileStoreBuilder``` is being created?

Reply:
Done

Reply:
Done